src/app/chat-list-item/chat-list-item.component.ts

```diff
@@ -33,7 +33,7 @@ export class ChatListItemComponent implements OnInit {
       this.imageUrl = data?.groupImageUrl;
       this.chatTitle = data?.groupName;
       /* Subscribe to the recipient only once by checking the isRecipientSubscribed */
-      if (!this.isRecipientSubscribed && data.type == 'dm') {
+      if (!this.isRecipientSubscribed && data && data.type == 'dm') {
```

Comment:
Why do we check if data is valid? Doesn't it return a chatMetadata on success response?

Reply:
I am getting an error when first starting up the app, if this is fine then it is not needed

Reply:
Changed it back, and tested again, no errors. Do not know why it gave me an error before.